ph5/core/ph5api.py

```diff
@@ -1397,15 +1397,31 @@ def receiver_cut(self, event_t_name, array_t, length):
 
     def get_extent(self, das, start=None, end=None, component=None):
```

Comment:
Thanks Nick. I'll fix this. I think originally I was going to make component optional, but was looking at other methods in PH5 that extent needs and it would take more changes than I have time for now. I will change this to a parameter instead of a keyword argument.